tests/Browser/ManageAccountsTest.php

```diff
@@ -12,16 +12,16 @@ class ManageAccountsTest extends DuskTestCase
     public function testView()
     {
         $this->browse(function (Browser $browser) {
-            $browser->visit(route('manage-accounts'))
-                ->screenshot('manage-accounts')
-                ->assertUrlIs(env('APP_URL') . __('route.manage_accounts'));
+            $browser->visit(route('manage.accounts.index'))
+                ->assertUrlIs(config('APP_URL') . __('route.manage_accounts'));
```

Comment:
Er is geen APP_URL in app.php

Reply:
Op line 58 staat het er bij mij wel

Reply:
Oh wait, er is url niet APP_URL. Ik ga het aanpassen!